topics/index.md

```diff
@@ -50,7 +53,8 @@ Administration
 * [Signals Handling](signals.md): How Valkey handles signals.
 * [Connections Handling](clients.md): How Valkey handles clients connections.
 * [Sentinel](sentinel.md): Valkey Sentinel is one of the official high availability deployment modes.
-* [Releases](releases.md): Valkey's development cycle and version numbering.
+* [Releases and Versioning](releases.md): Valkey's development cycle and version numbering.
+* [License](license.md): License information for Valkey.
```

Comment:
should we move license.md to the top before installation? release and versioning sounds like something first time users would like to read too, maybe before "release downloads"?